File: Nodejs/sequelize-and-node/models/language.js
'use strict';
const {
  Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
  class Language extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      // define association here
      models.Language.belongsTo(models.User, {
        foreignKey: "userId",
        as: "user",
      });
    }
  };
  Language.init({
    title: DataTypes.STRING,
    speak: DataTypes.FLOAT,
    understand: DataTypes.FLOAT,
    read: DataTypes.FLOAT,
    slug: DataTypes.STRING,
    userId: DataTypes.INTEGER,
  }, {
    sequelize,
    modelName: 'Language',
  });
  return Language;
};

File: Asp.net core/Aspcore/Aspcore/Models/Language.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Aspcore.Models
{
    public class Language
    {
        [Key]
        public int id { get; set; }
        public User user { get; set; }
        [Required]
        public int userId { get; set; }
        [Required]
        public string slug { get; set; }
        [Required]
        public string title { get; set; }
        [Required]
        public double speak { get; set; }
        [Required]
        public double read { get; set; }
        [Required]
        public double understand { get; set; }
        public DateTime createdAt { get; set; }
        public DateTime updatedAt { get; set; }
    }
}
